ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_creation.py

```diff
@@ -923,3 +923,28 @@ def test_jax_numpy_compress(
         a=arr[0],
         axis=ax,
     )
+    
+    
+@handle_frontend_test(
+    fn_tree="jax.numpy.iterable",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes(), shared_dtype=True
```

Comment:
We are generating only one array no need for `shared_dtype` and inside the `get_dtypes`  function you will have to specify the kind of dtypes to generate

Reply:
done!